### ISSUE TITLE
Disabled checkboxes should not be interactive with keyboard

### ISSUE THREAD

**Issue by zepumph** [phetsims/sun#519]
From https://github.com/phetsims/gravity-force-lab/issues/110 and potentially related to https://github.com/phetsims/sun/issues/517. If you go to GFL with a screen reader on, pressing space with toggle "checked" "not checked," but it won't toggle the underlying Property. I'm not sure where the discrepancy is.

**Comment by terracoda** [comment]
@zepumph, I think there is an `aria-disabled=true/false` or possibly even an HTML attribute that disables controls. We need to invoke an attribute on the second checkbox when the first one is unchecked so that if a user tabs to the second one, they hear something like:
- "Scientific Notation, checkbox, disabled".

I'll look up the attributes and see if we need the ARIA one or if the HTML5 one is well supported.

**Comment by terracoda** [comment]
For HTML5 see: https://www.w3.org/TR/html52/disabled-elements.html#disabled-elements
For ARIA see: https://www.w3.org/TR/wai-aria-1.1/#aria-disabled
For Authoring Practices see: https://www.w3.org/TR/wai-aria-1.1/#aria-disabled

@zepumph, if I am reading the can I use website correctly, I think we can use the HTML5 `disabled` attribute.

See: https://caniuse.com/#search=disabled

**Comment by zepumph** [comment]
@jessegreenberg and I did some research, searching for a way to may the checkbox un-interactive (essentially read-only) without removing it from the tab navigation order. We found https://stackoverflow.com/questions/12267242/how-can-i-make-a-checkbox-readonly-not-disabled/12267294 and went with a solution in there, using `onclick="return false"` to short circuit the default logic. We couldn't get other solutions to work like:
* readonly attribute
* event.preventDefault
* aria-disabled on its own

We also then added aria-disabled to make it more clear when the checkbox was disabled. Fuzzing is clear and the changeset is pretty small. I think this is solved.
